test/eventHandlers/msg.ts

```diff
@@ -21,7 +21,7 @@ function stackTraceRegExpProps(responseName: string, message: string): RegExpPro
 function workerMetadataRegExps(responseName: string) {
     return {
         [`${responseName}.workerMetadata.runtimeVersion`]: /^[0-9]+\.[0-9]+\.[0-9]+$/,
-        [`${responseName}.workerMetadata.workerBitness`]: /^(x64|ia32|arm64)$/,
+        [`${responseName}.workerMetadata.workerBitness`]: /^(x64|x86|arm64)$/,
```

Comment:
I saw the link here that you posted earlier: https://nodejs.org/api/process.html#processarch and it seems like there are more possible values than just `x64`, `x86` and `arm64`. Are those the only ones that we expect Functions to be run on? Relatedly, is `ia32` the only one that needs to be "normalized"?

Reply:
Makes sense to me!